### PR TITLE
data: Nanak Bandyopadhyay

### PR DESCRIPTION
**Pull request by Nanak360**
## Changes proposed
add my data

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6145"></a>

### CODE REVIEW

**Review by iamravi1 — APPROVED** [review]
LGTM 🎉

File: data/Nanak360.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/NBandyopadhyay",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/nanakbandyopadhyay/",
+      "icon": "FaLinkedinIn"
```
Comment (ChinmayMhatre) [inline data/Nanak360.json:31]:
```suggestion
      "icon": "FaLinkedin"
```

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Looks good! One small change in the icon name

File: data/Nanak360.json (hunk 1 above)
Comment (Nanak360) [reply data/Nanak360.json:31]: Screenshot 2023-04-17 at 10 51 28 AM
I'm using this icon

**Review by khushalarora — APPROVED** [review]
Looks good, thanks :tada:

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"><a href="https://linkfree.eddiehub.io/docs/how-to-guides/editing"></a></p>

**Pull request merged**